src/SFA.DAS.ContentApi/Application/Queries/GetContentQuery/GetContentQueryHandler.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using SFA.DAS.ContentApi.Data;
+
+namespace SFA.DAS.ContentApi.Application.Queries.GetContentQuery
+{
+    public class GetContentQueryHandler : IRequestHandler<GetContentQuery, string>
+    {
+        private readonly Lazy<ContentApiDbContext> _db;
+
+        public GetContentQueryHandler(Lazy<ContentApiDbContext> db)
+        {
+            _db = db;
+        }
+
+        public async Task<string> Handle(GetContentQuery request, CancellationToken cancellationToken)
+        {
+            await Task.Delay(0, cancellationToken);
```

Comment:
You could use Task.FromResult in the returns from this method and remove the async keyword.

Reply:
This will be changed once the DB call is added so it's temporary